docs/04-shortcuts-settings.md

```
@@ -0,0 +1,331 @@
+```
+
+## Extensions پیشنهادی
+
+### برای JavaScript/TypeScript
+- ESLint
+- Prettier
+- Path Intellisense
+- Auto Rename Tag
+- JavaScript (ES6) code snippets
+
+### برای Python
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)